### ISSUE TITLE
Yosemite support

### ISSUE THREAD

**Issue by ninjabong** [osxc/legacy-common#38]
Finally got around to installing Yosemite in my test VM to check out osxc and... not working.

[Ansible's facts are the problem](https://github.com/ansible/ansible/pull/8171), to work around the issue for now I did a git clone of ansible's current code (1.7) and things are working again.

<code>git clone https://github.com/ansible/ansible.git ~/src/github.com/ansible/ansible</code>
<code>source ~/src/github.com/ansible/ansible/hacking/env-setup</code>

Then you can proceed with ansible-playbook as expected.

@rricard - maybe, time to update ansible's version?  pip -upgrade ansible only got me to 1.6.10 so had to git clone to get 1.7, though perhaps that'll be finished before Yosemite is public.

**Comment by ninjabong** [comment]
The dock role is currently broken, due to dockutil being broken in Yosemite, but [a fix is already submitted](https://github.com/MoOx/dockutil/commit/8e8961ac8a65c2dc8812d6c2ddafc2d56b4831ce) so that should be working again soon.

**Comment by rricard** [comment]
Yep, that was to expect. I think dockutil should disappear for some time, the time we figure out the right tool for that. But we should indeed upgrade ansible. I'll try to do my best to find some more time !

**Comment by ninjabong** [comment]
The issue with dockutil is just a problem with the os versioning, ie: 10.# vs 10.##, so not that big of a deal.  While it's somewhat finicky at times it's the only tool I'm aware of to manipulate the dock from cli.

Ansible seems like just wait for 1.7 to be released, so that can still be put off for a little while 8)

**Comment by ninjabong** [comment]
Well Ansible 1.7 is out and works fine, dockutil sadly still hasn't merged the PR to fix 10.10 compatibility.

So for now to get a working ansible and osxc in a clean 10.10 install I'm doing:

```
sudo easy_install pip
sudo pip install ansible (fails but prompts to install xcode tools)
sudo pip install ansible (it'll work this time!)
ansible --version (to make sure it's 1.7 or higher)
git clone https://github.com/osxc/xc-common ~/src/github.com/osxc/xc-common
git clone https://github.com/ninjabong/xc-custom ~/src/github.com/ninjabong/xc-common
```

and then things have been working,

**Comment by rricard** [comment]
Yup great ! I finally got around this issue too.
That means xc-common is compatible, xc-boot of course needs to be refreshed again ...

**Comment by ninjabong** [comment]
[dockutil still hasn't had the patch to handle the new 10.10 versioning merged](https://github.com/kcrawford/dockutil/issues/25) so I've been using a new [dock1010](https://github.com/ninjabong/xc-custom/tree/master/roles/dock1010) role in my xc-custom to handle patching it for yosemite.  it checks for 10.10 and then downloads the patched version, otherwise uses the existing role so can be easily merged if dockutil remains unpatched after Yosemite's official release.

I took a quick glance at xc-boot last week and it seems like all that's really needed is to update [line 108](https://github.com/osxc/xc-boot/blob/master/boot.sh#L108) in boot.sh to pull version 1.7, but I've just been doing `pip install ansible` and skipping xc-boot entirely so might be something else I missed.
